zh_cn/security/index.md

```diff
@@ -8,7 +8,10 @@ lang: zh_cn
 
 ## 报告安全漏洞
 
-安全漏洞应该通过电子邮件报告到security@ruby-lang.org。这是一个非公开的邮件列表。 报告的问题将在被修正以后才会公开发布。
+安全漏洞应该通过电子邮件报告到security@ruby-lang.org ([the PGP public key](/security.asc))。这是一个非公开的邮件列表。 报告的问题将在被修正以后才会公开发布。
+
+这个邮件列表的成员为Ruby提供支持(包括Ruby代码提交者, 其他Ruby实现的作者, 发布者, 以及PaaS 平台提供者)。
```

Comment:
I think this line need the space between Chinese character and English character too.